GlotPress/GlotPress-WP
# Solve some warnings with the glossary suffixes

## Problem

We are getting some warnings with the glossary suffixes:

```
PHP Warning:  Undefined array key "abbreviation" in wp-content/plugins/GlotPress/gp-templates/helper-functions.php on line 409
PHP Warning:  foreach() argument must be of type array|object, null given in wp-content/plugins/GlotPress/gp-templates/helper-functions.php on line 409
```



## Solution

This PR:
- removes all elements in the `$suffixes` array with empty values.
- skip computing the `$suffixes[ $type ]` with empty values.

## Testing Instructions

- Translate some strings with the `develop` version and see some warnings in the `debug.log` file.
- Checkout to the branch of this PR. Now you should not see the previous warnings.
